<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="st" stdset="0">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QPlainTextEdit" name="temp_champ">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>132</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Température du champ : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>128</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Humidité du champ      :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>128</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Température de la serre :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Humidité de la serre      :   </string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="humid_champ">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="temp_serre">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="humid_serre">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>240</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>°C</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>°C</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="connecter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>350</y>
         <width>671</width>
         <height>101</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton1">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>90</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton4">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>240</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>136</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vitesse                             :</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="valeur_consigne">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="write_bouton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>300</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="read_bouton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>300</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Lire</string>
       </property>
      </widget>
      <widget class="QLabel" name="valeur_lue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
